huaweicloud/services/acceptance/cce/resource_huaweicloud_cce_cluster_test.go

```diff
@@ -225,28 +224,28 @@ func testAccCheckCCEClusterV3Destroy(s *terraform.State) error {
 
 		_, err := clusters.Get(cceClient, rs.Primary.ID).Extract()
 		if err == nil {
-			return fmtp.Errorf("Cluster still exists")
+			return fmt.Errorf("Cluster still exists")
 		}
 	}
 
 	return nil
 }
 
-func testAccCheckCCEClusterV3Exists(n string, cluster *clusters.Clusters) resource.TestCheckFunc {
+func testAccCheckClusterExists(n string, cluster *clusters.Clusters) resource.TestCheckFunc {
 	return func(s *terraform.State) error {
 		rs, ok := s.RootModule().Resources[n]
 		if !ok {
-			return fmtp.Errorf("Not found: %s", n)
+			return fmt.Errorf("Not found: %s", n)
 		}
 
 		if rs.Primary.ID == "" {
-			return fmtp.Errorf("No ID is set")
+			return fmt.Errorf("No ID is set")
 		}
 
 		config := acceptance.TestAccProvider.Meta().(*config.Config)
 		cceClient, err := config.CceV3Client(acceptance.HW_REGION_NAME)
 		if err != nil {
-			return fmtp.Errorf("Error creating HuaweiCloud CCE client: %s", err)
+			return fmt.Errorf("Error creating HuaweiCloud CCE client: %s", err)
```

Comment:
```suggestion
			return fmt.Errorf("error creating CCE client: %s", err)
```

Reply:
the message content will be modified in another PR.

Reply:
done.